modules/serverless-ossm-setup.adoc

```diff
@@ -177,6 +178,172 @@ spec:
+      numPartitions: <num_partitions>
+      replicationFactor: <replication_factor>
+  sink:
+      enabled: true
+  workloads: <2>
+  - name: kafka-controller
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-broker-receiver
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-broker-dispatcher
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-channel-receiver
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-channel-dispatcher
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-source-dispatcher
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-sink-receiver
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+----
+<1> The Apache Kafka cluster URL, for example: `my-cluster-kafka-bootstrap.kafka:9092`.
+<2> Enables sidecar injection for Knative Kafka pods.
+
+:FeatureName: The Knative Eventing integration with {SMProductShortName}
+include::snippets/technology-preview.adoc[leveloffset=+4]
+
+. Apply the `KnativeKafka` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Install `ServiceEntry` to make {SMProductName} aware of the communication between `KnativeKafka` components and an Apache Kafka cluster:
++
+[source,yaml]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: kafka-cluster
+  namespace: knative-eventing
+spec:
+  hosts: <1>
+    - <bootstrap_servers_without_port>
+  exportTo:
+    - "."
+  ports: <2>
+    - number: 9092
+      name: tcp-plain
+      protocol: TCP
+    - number: 9093
+      name: tcp-tls
+      protocol: TCP
+    - number: 9094
+      name: tcp-sasl-tls
+      protocol: TCP
+    - number: 9095
+      name: tcp-sasl-tls
+      protocol: TCP
+    - number: 9096
+      name: tcp-tls
+      protocol: TCP
+  location: MESH_EXTERNAL
+  resolution: NONE
+----
+<1> The list of Apache Kafka cluster hosts, for example: `my-cluster-kafka-bootstrap.kafka`.
+<2> Apache Kafka cluster listeners ports.
+
++
+[NOTE]
+====
+The listed ports in `spec.ports` are example TCP ports and depend on how the Apache Kafka cluster is configured.
+====
+
+. Apply the `ServiceEntry` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
++
+[IMPORTANT]
+====
+Ensure that addresses are set in the ServiceEntry when the protocol is not set or the protocol is set to TCP. When addresses are not set, all the traffic on the port specified in the ServiceEntry will be the same, no matter who the host is.
```

Comment:
```suggestion
Ensure that addresses are set in the ServiceEntry. If addresses is not set, all traffic on the port defined in the ServiceEntry is matched, regardless of the host.
```

Reply:
I'd use this text, we know protocol is TCP

Reply:
@pierDipi Added your sugestions!